cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -136,6 +136,27 @@ def password(self):
             password = self.scratch_info["password"]
         return password
 
+    @property
+    def email_address(self):
+        email_address = self.config.get("email_address")
```

Comment:
No strong opinion but maybe we should call it `admin_email` to better match the name in scratch org definition files.

Reply:
I find the SFDX-introduced nomenclature rather counterintuitive. On the theory that most of the time the value will come from Git, and secondly from `cumulusci.yml`, I'd rather call it what the user would understand it to be ("my user's email address").